ironfish-cli/src/commands/wallet/multisig/sign.ts

```diff
@@ -0,0 +1,101 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CurrencyUtils, Transaction } from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import { IronfishCommand } from '../../../command'
+import { RemoteFlags } from '../../../flags'
+
+export class MultiSigSign extends IronfishCommand {
+  static description = 'Aggregate signature shares from participants to sign a transaction'
+  static hidden = true
+
+  static flags = {
+    ...RemoteFlags,
+    account: Flags.string({
+      char: 'f',
+      description: 'Account to use when aggregating signature shares',
+      required: false,
+    }),
+    unsignedTransaction: Flags.string({
+      char: 'u',
+      description: 'Unsigned transaction',
+    }),
+    signingPackage: Flags.string({
+      char: 'p',
+      description: 'Signing package',
+    }),
+    signatureShare: Flags.string({
+      char: 's',
+      description: 'Participant signature share',
+      multiple: true,
+    }),
+    broadcast: Flags.boolean({
+      default: true,
+      allowNo: true,
+      description: 'Broadcast the transaction after signing',
+    }),
+  }
+
+  async start(): Promise<void> {
+    const { flags } = await this.parse(MultiSigSign)
+
+    const unsignedTransaction =
+      flags.unsignedTransaction?.trim() ??
+      (await CliUx.ux.prompt('Enter the unsigned transaction', { required: true }))
+
+    const signingPackage =
+      flags.signingPackage?.trim() ??
+      (await CliUx.ux.prompt('Enter the signing package', { required: true }))
+
+    let signatureShares = flags.signatureShare
+    if (!signatureShares) {
+      const input = await CliUx.ux.prompt('Enter the signature shares separated by commas', {
+        required: true,
+      })
+      signatureShares = input.split(',')
+    }
+    signatureShares = signatureShares.map((s) => s.trim())
+
+    const client = await this.sdk.connectRpc()
+
```

Comment:
Should we just handle this in RPC with an optional account and try to use default account if not passed?

Reply:
Yeah we can move it there